pombola/south_africa/templates/menu_entries.html

```diff
@@ -1,3 +1,4 @@
+{% load staticfiles %}
```

Comment:
Is this not already done in the parent template?

Reply:
throws an error when declared in the parent only, strangely just works when declared in the child. I also observed the same in other areas of the repo.

```TemplateSyntaxError at /
Invalid block tag on line 4: 'static'. Did you forget to register or load this tag?```